src/serializers/decoders/current/v16/decode_graph_entities.c

```
@@ -0,0 +1,496 @@
+
+// decode tensors
+static uint64_t _DecodeTensors
+(
+	SerializerIO rdb,  // RDB
+	GraphContext *gc,  // graph context
+	RelationID r       // relationship type
+) {
+	// format:
+	// edge format:
+	//     edge id
+	//     source node id
```

cppcheck

[error] Common realloc mistake

(memleakOnRealloc)